Debian/bin/lib/Cyrus/Docker/Command/distcheck.pm

```diff
@@ -0,0 +1,48 @@
+use v5.36.0;
+
+package Cyrus::Docker::Command::distcheck;
+use Cyrus::Docker -command;
+
+use Process::Status;
+use Term::ANSIColor qw(colored);
+
+my sub run (@args) {
+  say "running: @args";
+  system(@args);
+  Process::Status->assert_ok($args[0]);
+}
+
+sub abstract { 'do a distcheck against cyrus-imapd' }
+
+sub opt_spec {
+  return (
+    [ 'jobs|j=i',    'specify number of parallel jobs (default: 8) to run for make',
+                     { default => 8 },
+    ],
+  );
+}
+
+sub execute ($self, $opt, $args) {
+  my $root = $self->app->repo_root;
+  chdir $root or die "can't chdir to $root: $!";
+
+  # Assert that we're git clean?
```

Comment:
yeah, good idea

Reply:
I'll do this.  I think it makes even more sense now, with some changes subsequent to when I wrote that line: we want the commit sha that gets baked into the version to be accurate!

Reply:
I ended up adding `--dirty` instead, which doesn't just mean "build even if dirty" but "build ONLY if dirty".  Mostly this won't matter, but it means that you can reliably know that the archive name will include "-dirty" as per `tools/git-version.sh`